README.md

```diff
@@ -3,49 +3,86 @@
 [![CircleCI](https://circleci.com/gh/freedomofpress/signal-protocol.svg?style=svg)](https://circleci.com/gh/freedomofpress/signal-protocol)
 [![PyPI version](https://badge.fury.io/py/signal-protocol.svg)](https://badge.fury.io/py/signal-protocol)
 
-Experimental Python bindings to [`libsignal-client`](https://github.com/signalapp/libsignal-client) Rust signal protocol implementation. This project provides a Rust extension using [PyO3](https://pyo3.rs/) to define a `signal_protocol` Python module. See [here](https://cryptography.io/en/latest/limitations.html) for a fundamental limitation storing secrets in Python-allocated memory. ⚠️USE AT YOUR OWN RISK!⚠️
+Experimental Python bindings to Rust signal protocol implementation [`libsignal-client`](https://github.com/signalapp/libsignal-client).
+This project provides a Rust extension using [PyO3](https://pyo3.rs/) to define a `signal_protocol` Python module.
+See [here](https://cryptography.io/en/latest/limitations.html) for a fundamental limitation storing secrets in Python-allocated memory.
 
-# Installation
+⚠️USE AT YOUR OWN RISK!⚠️
 
-To use the wheel distributions you do not need the Rust toolchain installed:
+## Installation
+
+To use the wheel distributions you do not need the Rust toolchain installed.
+Simply run
 
 ```
 pip install signal-protocol
 ```
 
-# Usage
+## Usage
+
+### Initial client setup
+
+The following shows how to use this library to initialize a new Signal client.
+This is the first step that must be completed before the protocol can begin.
+
+For an overview of the Signal protocol, see [this blog post](https://www.redshiftzero.com/signal-protocol/).
```

Comment:
😅

Reply:
I think it's a really good post that fills an important gap in exposition of the protocol.